test/rules/disallow-padding-newlines-after-block-declarations.js

```diff
@@ -0,0 +1,65 @@
+var Checker = require('../../lib/checker');
+var assert = require('assert');
+
+describe('rules/disallow-padding-newlines-after-block-declarations', function() {
+    var checker;
+
+    beforeEach(function() {
+        checker = new Checker();
+        checker.registerDefaultRules();
+        checker.configure({ disallowPaddingNewLinesAfterBlockDeclarations: true });
+    });
+
+    it('should report padding after block', function() {
+        assert(checker.checkString('if(true){}\n\nvar a = 2;').getErrorCount() === 1);
+    });
+    it('should report padding after nested block', function() {
```

Comment:
padding newline before each it. I know this is inconsistent elsewhere, but that's what we've decided on.